### PR TITLE
Consistent naming of "Description" and "Notes" for a task

### PR DESCRIPTION
**Pull request by Jigao-Zeng**
---
name: Pull Request to fix issue #190
about: change the "Notes" title to "Description" in the task.
title: 'Change the Notes to Description '
labels:
assignees: 'Jigao Zeng'
---

**Describe the issue**
When adding a task it asks for a description but when displaying a task it displays what was written under notes. This cause inconsistency in naming and can confuse user

**Describe the solution**
Change the title of "Notes" to "Description" to make it consistent.

### CODE REVIEW

File: src/components/current-task/CurrentTaskNotes.jsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ function CurrentTaskNotes(props) {
       }`}
     >
       <div className="current-task-notes__header">
-        <span className="current-task-notes__title">Notes</span>
+        <span className="current-task-notes__title">Discription</span>
```
Comment (dikunong) [inline src/components/current-task/CurrentTaskNotes.jsx]: Nit: Description, not Discription :)

**Review by dikunong — CHANGES_REQUESTED** [review]
Just a small typo to fix :)

**Review by dikunong — APPROVED** [review]
LGTM!

**Review by kcho9906 — APPROVED** [review]

**Review by bbqnvd — APPROVED** [review]
Quick and easy. Nice.

**Pull request merged**